### DISCUSSION TITLE
problem with pip install

### DISCUSSION THREAD

**Discussion by mpgn** [pythongssapi/python-gssapi#293] [General]
### What went wrong?

I have trouble to install version 1.7.3 =>
`Command 'krb5-config --libs gssapi' returned non-zero exit status 127.`

### How do we reproduce?

![image](https://user-images.githubusercontent.com/5891788/174397649-6b2cbc74-f4da-4cf4-b400-05f1378d134a.png)

**Comment by mpgn** [comment]
└─$ sudo apt install libkrb5-dev

**Comment by CravateRouge** [comment]
It would be great to have a message more verbose than that such as :

```
Command 'krb5-config --libs gssapi' returned non-zero exit status 127, 'libkrb5-dev' package is probably missing on your OS.
```

**Reply by jborean93** [reply]
While it may be possible to add something like this  or at least add a check to see if `krb5-config` is present rather than the mysterious exit code 127  it's hard to state the package name of `libkrb5-dev` as the name is highly dependent on the system you are running under. For example Debian/Ubuntu it's `libkrb5-dev` but on RHEL-ish hosts it's `krb5-devel`.

I don't want to say this won't be added as I do see improving the error messages would be nice, it's just not a simple thing to do.

**Reply by CravateRouge** [reply]
I agree with you, but then something like `libkrb5 library is probably missing on your OS` would be more useful than `krb5-config` when googling the name to find the right package for a specific distribution.
